pkg/azuredisk/controllerserver.go

```diff
@@ -67,7 +67,7 @@ var (
 const (
 	sourceSnapshot           = "snapshot"
 	sourceVolume             = "volume"
-	azureDiskCSIDriverName   = "azuredisk_csi_driver"
+	CSIDriverMetricPrefix    = "azuredisk_csi_driver"
```

Comment:
Why the name change? This seems to have caused some unnecessary churn.

Reply:
Tried to keep in the uniformity in naming the variable with v2. Also, the "DriverName" was creating a confusion with the DriverName variable meaning.

Reply:
Let's revert this change until after we integrate upstream to avoid unnecessary merge conflicts.

Reply:
Done